pkgs/by-name/ca/cacert/package.nix

```diff
@@ -23,7 +23,7 @@ let
     lib.concatStringsSep "\n\n" extraCertificateStrings
   );
 
-  srcVersion = "3.113";
+  srcVersion = "3.113.1";
```

Comment:
No corresponding hash update?

Reply:
Let the record show, that I used 

```
 nix-update --build --commit cacert --version 3.113.1
```

And this was the result. :melting_face: